cli/bench/space.go

```diff
@@ -0,0 +1,57 @@
+package bench
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+// Create specified benchmark space with formatting and primary index.
```

Comment:
The space is not specified to the function in any way.
`createSpace creates benchmark space with formatting and primary index.`